### PR TITLE
Fix writing on paper

### PR DESCRIPTION
**Pull request by Markolie**
Changes:
1) Fixes a space in adminwho.
2) Fixes being unable to write on paper. Fixes https://github.com/ParadiseSS13/Paradise/issues/2300.

### CODE REVIEW

File: code/modules/paperwork/paper.dm [hunk 1]
```diff
@@ -65,7 +65,7 @@
 		send_asset_list(user.client, S.assets)
 
 	var/data
-	if(!(user.say_understands(null, all_languages["Galactic Common"]) && !forceshow) || forcestars) //assuming all paper is written in common is better than hardcoded type checks
+	if((!user.say_understands(null, all_languages["Galactic Common"]) && !forceshow) || forcestars) //assuming all paper is written in common is better than hardcoded type checks
```
Comment (tigercat2000) [inline code/modules/paperwork/paper.dm]: wha

How is this fixing the issue here? It's just making it so-
.... Oh, I'm an idiot.

Comment (Alex6511) [inline code/modules/paperwork/paper.dm]: How did this compile like that? :S

Comment (Markolie) [inline code/modules/paperwork/paper.dm]: It's still a valid check. It just included forceshow in the negative check, which broke thing.

Comment (Alex6511) [inline code/modules/paperwork/paper.dm]: It had unbalanced parenthesis though, that should at least throw an error.

**Pull request merged**